docs/sources/configure-client/grafana-agent/java/_index.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Profiling Java via the Grafana Agent"
+menuTitle: "Profiling Java via the Grafana Agent"
+description: "Learn about using Grafana Agent for continuous profiling Java processes for performance optimization."
+weight: 20
+---
+
+# Profiling Java via the Grafana Agent
```

Comment:
By "via", do you mean "using"? If so, it might be clearer to swap the words. What do you think? I'll go ahead and make the suggestions in the frontmatter. 
```suggestion
# Profiling Java using the Grafana Agent
```